gflags/argument_parser.py

```diff
@@ -325,7 +325,7 @@ def __init__(self, list_sep):
 
   def Serialize(self, value):
     """Serialize a list as a string, if possible, or as a unicode string."""
-    output = cStringIO.StringIO()
+    output = io.StringIO()
     writer = csv.writer(output)
 
     # csv.writer doesn't accept unicode, so we convert to UTF-8.
```

Comment:
I believe there is a lurking bug on the following line when running in python3 (use of `unicode` keyword).

Instead, I believe you can replace all occurrences (there's only one) of `CsvListSerializer()` with `ListSerializer(',')`.  This prevents the need for importing the `io` module entirely.